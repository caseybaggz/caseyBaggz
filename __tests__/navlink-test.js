import React from 'react';
import NavLink from '../javascripts/components/NavLink';
import renderer from 'react-test-renderer';


it('should render NavLink component successfully', () => {
  const tree = renderer.create(<NavLink to="/works/lays" />).toJSON();
  expect(tree).toMatchSnapshot();
});
