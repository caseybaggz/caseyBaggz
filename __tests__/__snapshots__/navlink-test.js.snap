exports[`test should render NavLink component successfully 1`] = `
<a
  onClick={[Function bound handleClick]}
  style={Object {}} />
`;
